sys-libs/mesa/mesa-17.1.7.recipe

```diff
@@ -27,119 +28,106 @@ REQUIRES="
 PROVIDES_devel="
 	mesa${secondaryArchSuffix}_devel = $portVersion
 	devel:libGL$secondaryArchSuffix = $portVersion
-	devel:libEGL$secondaryArchSuffix = $portVersion
+	devel:libEGL$secondaryArchSuffix = 1.0.0
 	"
 REQUIRES_devel="
 	mesa$secondaryArchSuffix == $portVersion base
 	"
 
+SUMMARY_swpipe="The Mesa LLVM enhanced Gallium software pipe renderer"
+DESCRIPTION_swpipe="This 3D BGLRenderer add-on provides Gallium LLVM \
+enhanced software rendering. Software pipe rendering performs all \
+3D rendering on the systems CPU and doesn't require any specialized \
+hardware. The usage of LLVM over traditional rasterization gives this \
+renderer a boost in performance.
+
+Gallium software pipe rendering is in an extremely early state as of \
+this version of Mesa, and may not function as expected."
+
+PROVIDES_swpipe="
+	mesa${secondaryArchSuffix}_swpipe = $portVersion
+	"
+REQUIRED_swpipe="
+	mesa$secondaryArchSuffix == $portVersion base
+	lib:libLLVM_4.0$secondaryArchSuffix
+	"
+
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	devel:libz$secondaryArchSuffix
+	llvm${secondaryArchSuffix}
 	"
 BUILD_PREREQUIRES="
-	llvm${secondaryArchSuffix}
-	mako_python
-	cmd:bison
-	cmd:find
-	cmd:flex
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
 	cmd:gcc$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
-	cmd:scons
-	cmd:sed
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:which
 	"
 
 BUILD()
 {
 	if [ -n "$secondaryArchSuffix" ]; then
 		export HAIKU_SECONDARY_ARCH="$effectiveTargetArchitecture"
 	fi
-	scons $jobArgs build=release
-	#scons $jobArgs build=debug
+
+	autoreconf -fi
+	runConfigure ./configure \
+		--disable-dri \
+		--disable-glx \
+		--disable-glx-tls \
+		--disable-gbm \
+		--disable-gles1 \
+		--disable-gles2 \
+		--disable-shared-glapi \
+		--with-gallium-drivers=swrast \
```

Comment:
this doesn't look right.  swpipe ?

Reply:
or maybe softpipe / llvmpipe

Reply:
yeah, I did't commit with swpipe. Will add it.